src/flashbar/common.tsx

```diff
@@ -25,16 +23,12 @@ export function useFlashbar({
   onItemsRemoved?: (items: FlashbarProps.MessageDefinition[]) => void;
   onItemsChanged?: (options?: { allItemsHaveId?: boolean; isReducedMotion?: boolean }) => void;
 }) {
-  const { __internalRootRef } = useBaseComponent('Flashbar', {
-    props: { stackItems: restProps.stackItems },
-  });
   const allItemsHaveId = useMemo(() => items.every(item => 'id' in item), [items]);
   const baseProps = getBaseProps(restProps);
   const ref = useRef<HTMLDivElement | null>(null);
   const [breakpoint, breakpointRef] = useContainerBreakpoints(['xs']);
-  const mergedRef = useMergeRefs(ref, breakpointRef, __internalRootRef);
+  const mergedRef = useMergeRefs(ref, breakpointRef);
   const isReducedMotion = useReducedMotion(ref);
-  const isVisualRefresh = useVisualRefresh();
```

Comment:
Moved `isVisualRefresh` from the hook directly into component code to make DCE work for ALWAYS_VISUAL_REFRESH mode